config/event_handling.tmpl.yml

```diff
@@ -1444,6 +1454,18 @@ event_handling:
   NODE_NODENOTREADY:
   - refresh:
     - ems
+  NODE_NODESCHEDULABLE:
```

Comment:
@dkorn 
Event Switchboard PR #4328 which is merged this week has moved the action mappings in the event_handling: section of event_handling.tmpl.yml into the automate model. This section in the yaml file will not be honored any more.
I am going to make a PR to remove this section.
This new events handling for NODE should be added into automate model. We can walk through this together if you want to.